backend/translation/models.py

```diff
@@ -4,18 +4,21 @@
 
 from .metadata import LANGUAGE_CHOICES
 
+HUMAN_EDITED = "he"
+MACHINE_GENERATED = "mg"
```

Comment:
Just now thought about this: To keep these choice names consistent with that of transcript, shall we rename `"he"` as `"umg"` ?

We might also have to write a custom migration to modify all the records.

Reply:
Yeah, sure. I think we could change it. I will write the custom migration.